src/transformers/tokenization_utils.py

```diff
@@ -1785,7 +1911,7 @@ def _convert_encoding(
             if return_special_tokens_mask:
                 encoding_dict["special_tokens_mask"].append(e.special_tokens_mask)
             if return_offsets_mapping:
-                encoding_dict["offset_mapping"].append([e.original_str.offsets(o) for o in e.offsets])
+                encoding_dict["offset_mapping"] = [o for o in e.offsets]
```

Comment:
Should it be like above:
```suggestion
                encoding_dict["offset_mapping"].append(e.offsets)
```